ingest/src/main/scala/hydra.ingest/http/IngestionEndpoint.scala

```diff
@@ -90,44 +87,66 @@ class IngestionEndpoint[F[_]: Futurable](
     }
   }
 
+  //TODO: Need to figure out how to reset this gauge. Currently will only add to gauge
```

Comment:
Let's remove this for now and put this into a Github Issue instead. The Todos tend to get buried down the road.